docs/pre_release_features.md

```diff
@@ -46,30 +46,30 @@ If you'd like to review the changes for a nightly release, you can use the [comp
 
 ## Beta and Experimental Namespaces
```

Comment:
Change to "Beta and Experimental Features"?

Reply:
Yup